python/cuml/test/test_fil.py

```diff
@@ -202,11 +202,17 @@ def test_fil_skl_classification(n_rows, n_columns, n_estimators, max_depth,
                                 n_classes, storage_type, model_class):
     # skip depth 20 for dense tests
     if max_depth == 20 and not storage_type:
-        return
+        pytest.skip('Skipping depth 20 for dense tests')
 
     # FIL not supporting multi-class sklearn RandomForestClassifiers
     if n_classes > 2 and model_class == RandomForestClassifier:
-        return
+        pytest.skip('FIL does not support multi-class sklearn ' +
+                    'RandomForestClassifier')
+
+    # When n_classes=25, fit a single estimator only
+    if n_classes == 25 and n_estimators > 1:
```

Comment:
Could we do this instead by explicitly listing the test cases in mark_parametrize? I think that would be about the same length but clearer. You can explicitly list relevant combos of n_classes, max_depth, n_estimators while still relying on a separate `mark_parametrize` line for storage_type if e.g. you want all of those repeated for both storage types (though I'm not sure we need full coverages of every variant n_classes and every storage type)

Reply:
Addressed in the latest commit.